src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -528,6 +528,56 @@ public SequencePosition GetPosition(long offset)
             return Seek(offset);
         }
 
+        /// <summary>
+        /// Translate opaque <see cref="System.SequencePosition"/> value to numerical position offset
+        /// </summary>
+        /// <param name="position"><see cref="System.SequencePosition"/> to translate to offset</param>
+        /// <returns>Returns the translated offset from the start of the sequence</returns>
+        /// <exception cref="System.ArgumentOutOfRangeException">Thrown when provided <see cref="System.SequencePosition"/> is invalid</exception>
```

Comment:
What makes a sequence position `invalid`? Can we clarify that in the docs. In some "invalid" cases, we don't throw `ArgumentOutOfRangeException`, correct?

Also, look at how exception messages tend to be worded. For example, it should be written like this:
```suggestion
        /// <exception cref="System.ArgumentOutOfRangeException">The provided <see cref="System.SequencePosition"/> is invalid.</exception>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions#how-to

https://docs.microsoft.com/en-us/dotnet/api/system.buffers.readonlysequence-1.-ctor?view=netcore-3.1#System_Buffers_ReadOnlySequence_1__ctor_System_Buffers_ReadOnlySequenceSegment__0__System_Int32_System_Buffers_ReadOnlySequenceSegment__0__System_Int32_

Reply:
Thank's for the links

Reply:
> What makes a sequence position invalid? Can we clarify that in the docs. In some "invalid" cases, we don't throw ArgumentOutOfRangeException, correct?

@ahsonkhan I did a check and in this method we only throw `ThrowHelper.ThrowArgumentOutOfRangeException_PositionOutOfRange()`(`ArgumentOutOfRangeException`) we validate only `SequencePosition` other exception should be bugs.

Reply:
Documentation updated